udp_driver/include/udp_manager_node.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef UDP_DRIVER__UDP_MANAGER_NODE_HPP_
+#define UDP_DRIVER__UDP_MANAGER_NODE_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <map>
+#include <memory>
+
+#include "udp_msgs/msg/udp_packet.hpp"
+#include "udp_msgs/srv/udp_send.hpp"
+#include "udp_msgs/srv/udp_socket.hpp"
+
+namespace udp_driver
+{
+
+class UdpManagerNode : public rclcpp::Node
+{
+  public:
+    UdpManagerNode();
+    ~UdpManagerNode(){}
```

Comment:
Empty destructor should be removed. Adding a destructor breaks move semantics and requires adding constructors/operators from the [rule of five](https://en.cppreference.com/w/cpp/language/rule_of_three) to fix them. Simply leaving out the destructor if it isn't used is better because it will be auto-generated with the default without breaking move semantics.

Reply:
good to know. I'll fix this